### PR TITLE
Add `release_tag` to config checklist

### PR DESCRIPTION
**Pull request by sjspielman**
Does what it says! I realized this was missing from the file. 

Worth noting this is the first time we're doing a workshop with the updated version of `training-specific-template` so there may be more of these small fixes. Please have a quick look [at the config](https://github.com/AlexsLemonade/training-specific-template/blob/main/_config.yaml) to check that nothing else is missing too - I think we're good now?

### CODE REVIEW

**Review by jashapiro — COMMENTED** [review]
I think this may be intentionally not in the list, as we don't update this until much later? The docker tag also probably doesn't get updated yet though...

I think the better thing to do might be to add the docker tag to https://github.com/AlexsLemonade/training-specific-template/blob/main/setup-issue-templates/workshop-release.md? 

We will also want to note there (or somewhere) that creating the tag will build a new docker image with a matching tag (this is automatic now!). Which may mean we don't really need a separate docker tag, but it is probably worth leaving them separate for now.

**Comment by sjspielman** [comment]
> I think the better thing to do might be to add the docker tag to https://github.com/AlexsLemonade/training-specific-template/blob/main/setup-issue-templates/workshop-release.md?

This makes sense to me

**Comment by sjspielman** [comment]
@jashapiro is this about what you had in mind?

File: setup-issue-templates/workshop-release.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
 In [`AlexsLemonade/training-modules`](https://github.com/AlexsLemonade/training-modules), [create a new release](https://github.com/AlexsLemonade/training-modules/releases/new) that ties that material to this specific workshop. We use the convention `<year>-<month>` for tags and `<year> <month>` for release titles for Data Lab workshops.
 
-Don't forget to update the `_config.yaml` file in _this repository_ to point to the correct release.
+Be sure to update the `_config.yml` file _in this repository_ accordingly:
+
+- `release_tag` and `docker_tag` should point to the correct release
+- `docker_repo` should, in most cases, be filled in with `training_rnaseq`
```
Comment (jashapiro) [inline setup-issue-templates/workshop-release.md:6]: The docker repo has been updated by the automated build process to be more uniform (and less topic-specific)
In practice, we could probably just fill this in by default.
```suggestion
- `docker_repo` should, in most cases, be filled in with `training_rstudio`
```

**Review by jashapiro — APPROVED** [review]
Could you also update the default in https://github.com/AlexsLemonade/training-specific-template/blob/a285574f61ffa5fd5426a8a795df710d9e3e9018/_config.yaml#L30-L31

**Comment by sjspielman** [comment]
> Could you also update the default in 

@jashapiro Did you want a default of `edge` or similar for `docker_tag`?

**Comment by jashapiro** [comment]
> > Could you also update the default in
> 
> @jashapiro Did you want a default of `edge` or similar for `docker_tag`?

You could. I don't think it matters too much.

**Comment by sjspielman** [comment]
> You could. I don't think it matters too much.

I did it for the pun. Worth it.

**Pull request merged**